services/ingestion/etl/fts/xls/src/lib/transform.js

```diff
@@ -0,0 +1,234 @@
+// @flow
+
+import countries from 'i18n-iso-countries';
+import type { Project } from '@eubfr/types';
+import getCountryCode from '@eubfr/lib/getCountryCode';
+import sanitizeBudgetItem from '@eubfr/lib/budgetFormatter';
+
+/**
+ * Gets country code from a country name.
+ * Contains a map of exceptions.
+ *
+ * @memberof FtsXlsTransform
+ * @param {String} countryName The name of the country
+ * @returns {String} The ISO 3166-1 country code
+ */
+const getCodeByCountry = countryName => {
+  // There are a few exceptions which should be normalized, regardless of the library
+  const corrections = new Map([
```

Comment:
I'm not sure for which part of the article you refer to, could you clarify?

Reply:
`When to use Map? And when to use Object?`

> Object is the great choice for scenarios when we only need simple structure to store data and knew that all the keys are either strings or integers (or Symbol), because creating plain Object and accessing Object’s property with a specific key is much **faster** than creating a Map (_literal vs constructor, direct vs get() function call — you know who wins already_).

The advantages of a `Map` are not pertinent in this context